src/config.js

```diff
@@ -1,44 +1,49 @@
-const { parseAsArray } = require('./lib');
+const { parseAsArray } = require('./utils');
+const adapters = require('./adapters');
 
 const configPath = () => (
   process.env.REACT_I18N || `${process.cwd()}/.i18nrc`
 );
 
-const readConfig = () => (
-  new Promise((resolve, reject) => {
-    try {
-      const config = require(configPath());
-      resolve(config);
-    } catch(e) {
-      reject(e);
-    }
-  }
-));
+const readServiceKey = (config) => {
+  if (typeof config.serviceKey !== 'string') return Promise.resolve(config);
+
+  return adapters.json.read(config.serviceKey).then((key) => {
+    config.serviceKey = key;
+    return config;
+  });
+};
 
 const readDefaultSettings = (config) => {
   const defaults = Object.assign({}, {
     output: './tmp',
     range: 'A1:M1000',
     locales: ['en-US'],
     filename: `i18n-${new Date().getTime()}.csv`,
-    type: 'module',
+    format: 'module',
```

Comment:
what are your thoughts on using `commonjs` rather than `module` for the default format & adapter? module makes sense but is a bit ambiguous and commonjs is the actual spec